scripts/data_ingestion/download_lahman_data.py

```
@@ -0,0 +1,122 @@
+
+
+def download_lahman_zip(output_dir: Path) -> Path:
+    """Download Lahman database zip file."""
+    zip_path = output_dir / "lahman_master.7z"
+    
+    if zip_path.exists():
+        print(f"Archive file already exists at {zip_path}")
+        return zip_path
+    
+    print(f"Downloading Lahman Baseball Databank from {LAHMAN_ZIP_URL}")
+    print(f"This may take a few minutes...")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)